### PR TITLE
Adding all the new predictions

### LINKED ISSUE #16: Create a dashboard for the new PLparameters.pdf
PLparameters.pdf has new predictions, we should show them in another dashboard. And perhaps show them also for the countries with old predictions as well, for comparison.

### PR DESCRIPTION
**Pull request by lukipuki**

### CODE REVIEW

File: covid_graphs/covid_graphs/formula.py [hunk 1]
```diff
@@ -31,19 +31,29 @@ def ATG_formula(TG, A, exponent, min_case_count=200):
...
+    It's the first day for which the number of cumulative active is more than
+    formula.min_case_count.
+    """
+    return np.argmax(cumulative_active > formula.min_case_count)
+
+
 class Curve():
-    """Class containing an curve derived from a formula.
+    """
+    Class containing an curve derived from a formula.
```
Comment (miskosz) [inline covid_graphs/covid_graphs/formula.py]: Nit: a curve

File: covid_graphs/covid_graphs/formula.py [hunk 2]
```diff
@@ -31,19 +31,29 @@ def ATG_formula(TG, A, exponent, min_case_count=200):
...
 
 
+def first_day_of_curve(cumulative_active, formula):
+    """
+    Returns the first day of the curve formed by 'formula'.
+
+    It's the first day for which the number of cumulative active is more than
+    formula.min_case_count.
+    """
+    return np.argmax(cumulative_active > formula.min_case_count)
```
Comment (miskosz) [inline covid_graphs/covid_graphs/formula.py:41]: Did you intentionally change `>` to `>=`?

File: data/download_all.zsh [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
 short_names["United Kingdom"]=UK
+short_names["New Zealand"]=NZ
 
 python3.7 -m venv --system-site-packages venv && source venv/bin/activate
 pip install $1/
-for country in Germany Iran Italy Spain "United Kingdom" US
+
+countries=("United Kingdom" Korea Austria Iceland Jordan Switzerland Croatia Australia Canada
+  Germany Israel Italy Malaysia "New Zealand" Spain Belgium Chile Czechia Latvia
+  Lithuania Netherlands Norway Portugal US)
```
Comment (miskosz) [inline data/download_all.zsh]: You are using `US` here and not `United States`, not sure if intentional.

**Comment by lukipuki** [comment]
FYI, there's something incorrect about the predictions - I found inconsistencies between us and Katka/Rišo for Croatia and Czechia. They are investigating. Until then, we shouldn't merge this.

**Review by miskosz — APPROVED** [review]
LGTM

**Comment by miskosz** [comment]
@lukipuki I'd suggest commenting Croatia and Czechia out and merging like that.

We are not displaying the graphs anyway, are we?

File: covid_graphs/covid_graphs/formula.py (hunk 2 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/formula.py:41]: Yes, that's how Katka does the fitting. I spotted in on Lithuania, where the difference is 12 days :smile:

**Comment by miskosz** [comment]
Oh, those two were just examples, weren't they :D

File: data/download_all.zsh (hunk 3 above)
Comment (lukipuki) [reply data/download_all.zsh]: Thanks for noticing, that's a bug.

**Comment by lukipuki** [comment]
Yes, those are just examples, I suspect a bigger issue somewhere (could be on either side).

But you are right, let's merge and we'll wait for their comments.

File: covid_graphs/covid_graphs/formula.py (hunk 1 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/formula.py]: Done

**Pull request merged**